### DISCUSSION TITLE
Q-Toggle custom colors

### DISCUSSION THREAD

**Discussion by mtn090899** [quasarframework/quasar#11545] [General - Components / Directives / etc] [answered]
Hi, new to utilizing the quasar framework. I am trying to have more customization for the quasar toggle component. I figured out how to utilize colors using hex-values but I want there to be a combination colors like shown below. 
 
![20211202_140232](https://user-images.githubusercontent.com/78068147/144486311-37e4220b-d3c3-4c30-8e3c-01f6b7259d94.jpg)

I've also been trying to get the unchecked q-toggle to be a certain color opposed to its default white and I cannot quite figure it out. 

Here is the code I have so far. Thank you. 

`<template>
    <q-toggle
        
        v-model="value"
        :size ="size"
        :disable ="disable"
        color = bluegrey-900
      
    />
</template>

<script>
    export default {
        data() {
            return {
                value : false
            }
        },
        props: {
            size:{
                type: String,
                default: 'lg'
            } ,
            disable: {
                type: Boolean,
                default: false
            } ,
        },
        
    }
</script>

<style scoped>

</style>`

**Comment by MilosPaunovic** [comment, marked as answer]
Hello and welcome to Quasar community!

First thing you want to check when using new componenet is the `API` section on top of the page - it contains a listing of all available props, slots and methods and has a search input to narrow your results.

So, for `QToggle`, you need [https://quasar.dev/vue-components/toggle#qtoggle-api](https://quasar.dev/vue-components/toggle#qtoggle-api).

There is a `color` prop which takes `Color name for component from the Quasar Color Palette`, so any color from [here](https://quasar.dev/style/color-palette#color-list). For the other part of your query, take a look at the `keep-color` prop.

And for any other style changes, you could try changing the CSS of the element.

**Reply by mtn090899** [reply]
I've already looked at everything you mentioned, thank you for the CSS style change suggestion. Do you have any clue where I can find what element I need to change? I try changing all of q-toggle, but thats not quite what Im looking for. Im looking to change the color of the bar and circle that makes up q-toggle. 

This does give me a basis to play with however, thank you!

**Reply by MilosPaunovic** [reply]
You should inspect element through browser console, to try and find a right one to apply your custom styling.
